app/views/guests/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= @guest.name %>
+<p> Rooms Stayed In:</p>
+<ul>
+<% @guest.rooms.each do |room| %>
+<li> <%= room.suite %> <%= room.id %> ($<%= room.rate %> at the <%= room.hotel.name %>) </li>
+<% end %>
+</ul>
+</div>
+
+<h2>Add another room</h2>
+<%= form_with url: "/guest_rooms/#{@guest.id}", method: :post, local: true do |f| %>
```

Comment:
This is not a RESTful route. Currently the way that the route is formatted it looks something like this:

```ruby
post "/guest_rooms/:id"
```

This does not follow RESTful convention as the path for a POST never ends in an ID.

If you did someting like: `url: "/guest_rooms"` then that would be ok, but you would need to add a hidden field to the form where you put the guests ID.

```ruby
<%= f.hidden_field :guest_id, value: @guest.id %>
```